resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,18 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if (isset($opengraph))
+    <meta property="og:site_name" content="{{ 'osu! » '.$opengraph['site_name'] ?? 'osu!' }}">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{ strtok(Request::url(), '?') }}">
```

Comment:
This should use `$canonicalUrl` (also used elsewhere) and check in the `isset` above.